source/includes/_rars_top_referrers.md

```diff
@@ -0,0 +1,159 @@
+### **Top Referrers**
+<a name="top_referrers"></a>
+
+Describes the distribution of referrers for an advertiser for a given time period as well as attributing the source
+of the referral.  This API endpoint is the data source for the Traffic Detail Report report in client center.
+
+### Resource Overview&nbsp;
+
+| Method | URI Format |
+|---|---|
+| GET | /client_reports/top_referrer/[gmaid]?[query_params] |
+
+### Parameters&nbsp;
+
+When using the GET method, the results can be filtered using these parameters:
+
+| Param     | Required | Function |
+|-----------|-----|---|
+| start_date | Yes | Restricts the results to those occurring on or after this date and time. |
+| end_date | Yes | Restricts the results to those occurring on or before this date and time. |
+| gmaid | Yes | Restrict results to the specified advertiser |
+|interval_size|No|Return subtotals for the given intervals within the start-end time provided. Accepts calendar_month, calendar_week, or day (Default is day)|
```

Comment:
@Jordan3900 we are not using interval_size in the report building process. So it should not be a param here.